docs/catalog.md

```diff
@@ -103,7 +103,11 @@ If you're not using OpenWhisk in Bluemix or if you want to set up your Cloudant
 
 ### Listening for changes to a Cloudant database
 
-You can use the `changes` feed to configure a service to fire a trigger on every change to your Cloudant database.
+You can use the `changes` feed to configure a service to fire a trigger on every change to your Cloudant database. The parameters are as follows:
+
+- `dbname`: Name of cloudant database.
+- `includeDoc`: If set to true, each trigger event that is fired includes the modified Cloudant document. 
+- `maxTriggers`: Stop firing triggers when this limit is reached. Defaults to 1000.
```

Comment:
pls also document the maximum of 10.000. Its missed in the alarms package as well.